macro-pdfviewer-webjar/pom.xml

```
@@ -94,8 +94,9 @@
                   <value>$0;
   const queryString = document.location.search.substring(1);
   const params = (0, _ui_utils_js__WEBPACK_IMPORTED_MODULE_0__.parseQueryString)(queryString);
-  if (params.trustedorigins !== undefined &amp;&amp; params.trustedorigins !== '') {
-    HOSTED_VIEWER_ORIGINS.push.apply(HOSTED_VIEWER_ORIGINS, params.trustedorigins.split(',')) }</value>
+  const trustedOrigins = params.get('trustedorigins');
```

```suggestion
  const trustedOrigins = (params.get && params.get('trustedorigins')) || params.trustedorigins;
```

Its weird that it changed like this. Did we update pdf viwer.js recently? The change can come from this or it might have to do with the browser or other dependencies. In this case, it would be better to take caution